android/src/main/res/layout/player_controls.xml

```diff
@@ -48,6 +48,7 @@
             android:textColor="@android:color/white"
             android:textSize="16sp"
             android:textStyle="bold"
+            android:visibility="gone"
```

Comment:
based on the close button constraints, we can use invisible here to keep the existing controls positioning